rasa_core/training/online.py

```diff
@@ -469,8 +469,8 @@ def _write_stories_to_file(export_file_path, sender_id, endpoint):
 
 
 def _predict_till_next_listen(endpoint,  # type: EndpointConfig
-                              sender_id  # type: Text
-                              ):
+                              sender_id,  # type: Text
+                              finetune): # type: Boolean
```

Comment:
type needs to come directly after the argument (`):`) needs to go onto the next line